### DISCUSSION TITLE
Make table filled with data from array of objects

### DISCUSSION THREAD

**Discussion by Linux-Alex** [orchidsoftware/platform#2482] [Q&A]
Hi,
I'm doing a WooCommerce manager and I'm retriving data from shops with this [Laravel WooCommerce Package](https://codexshaper.github.io/docs/laravel-woocommerce/). I get data formated in an array of the same object. 

How can I output this array into a table like this:
![Example](https://orchid.software/img/layouts/table.png)

Please help

**Comment by tabuna** [comment]
Without an example of the data, we can only guess.
Look at an example of a table where the values are wrapped in a `Repository` object
